src/core/shards/core.hpp

```diff
@@ -1682,9 +1696,13 @@ struct TableDecl : public VariableBase {
 
   SHExposedTypesInfo exposedVariables() { return SHExposedTypesInfo(_exposedInfo); }
 
-  static SHTypesInfo inputTypes() { return CoreInfo::AnyType; }
+  static SHOptionalString help() { return SHCCSTR("Creates an empty table."); }
 
-  static SHTypesInfo outputTypes() { return CoreInfo::AnyType; }
+  static SHTypesInfo inputTypes() { return CoreInfo::NoneType; }
+  static SHOptionalString inputHelp() { return SHCCSTR("N/A"); }
+
+  static SHTypesInfo outputTypes() { return CoreInfo::NoneType; }
```

Comment:
Why `None`?

Reply:
Reverted this change (rebased and force-pushed to this commit).